### PR TITLE
Auto Routine Changes

### PR DESCRIPTION
**Pull request by Lu-han-wang**
**Options: **
- center speaker one-piece auto(doesn't cross auto line)
- center two piece auto(one preload and pickup the center one, cross the auto line)
- one piece amp auto(cross the auto line, not likely to be in the way of picking up the side note)
- one piece left speaker auto(shoot one preloaded note and get out of the way)
- one piece right speaker auto(shoot one preloaded note and get out of the way)

### CODE REVIEW

File: src/main/java/org/jmhsrobotics/frc2024/subsystems/intake/commands/IntakeCommand.java [hunk 1]
```diff
@@ -40,7 +40,7 @@ public void initialize() {
 	@Override
 	public void execute() {
 		this.intakeSubsystem.set(this.speed);
-		this.shooterSubsystem.set(-.05, ControlType.VOLTAGE);
+		this.shooterSubsystem.set(-0.02, ControlType.VOLTAGE);
```
Comment (m10653) [inline src/main/java/org/jmhsrobotics/frc2024/subsystems/intake/commands/IntakeCommand.java:43]: Why do we have this line here. 0.02 volts is not going to be enough the move the motor. the range should be +-13 volts ish. So 10% power would be like 1.3 volts.

File: src/main/java/org/jmhsrobotics/frc2024/controlBoard/CompControl.java [hunk 2]
```diff
@@ -24,7 +24,7 @@ private boolean isTurbo() {
 
 	@Override
 	public double xInput() {
-		return this.driver.getLeftX() * (isTurbo() ? 1.0 : slowMode);
+		return this.driver.getLeftX() * (isTurbo() ? 1.0 : slowMode) * 0.7;
 	}
```
Comment (m10653) [inline src/main/java/org/jmhsrobotics/frc2024/controlBoard/CompControl.java:29]: I think this change was meant to be limiting the rotation speed not the x speed. With this change it makes X rotation 30% slower. That explains why the driver was not seeing an effect last practice.

File: src/main/java/org/jmhsrobotics/frc2024/Constants.java [hunk 3]
```diff
@@ -85,7 +85,7 @@ public static class SwerveConstants {
 		/* START DRIVER CONFIGURABLES */
 
 		// configure to change drivetrain turbo mode
-		public static final double kMaxSpeedFeetPerSecond = 14;
+		public static final double kMaxSpeedFeetPerSecond = 16;
 
 		// configure to change drivetrain slow mode
 		public static final double slowSpeedFeetPerSecond = 12; // TODO: Remove used when not in turbo mode move to Comp
```
Comment (m10653) [inline src/main/java/org/jmhsrobotics/frc2024/Constants.java:91]: Was slow mode also requested to be increased?

**Review by m10653 — CHANGES_REQUESTED** [review]

File: src/main/java/org/jmhsrobotics/frc2024/Constants.java (hunk 3 above)
Comment (Lu-han-wang) [reply src/main/java/org/jmhsrobotics/frc2024/Constants.java:91]: i'll confirm it with Annabelle today

File: src/main/java/org/jmhsrobotics/frc2024/subsystems/intake/IntakeSubsystem.java [hunk 4]
```diff
@@ -56,10 +57,8 @@ public void periodic() {
 		// SmartDashboard.putBoolean("Intake/lowSwitchState",
 		// this.lowSwitch().isPressed());
 
-		// SmartDashboard.putNumber("Intake/lowerSensorReading",
-		// this.lowerSensor.getRange());
-		// SmartDashboard.putNumber("Intake/upperSensorReading",
-		// this.upperSensor.getRange());
+		SmartDashboard.putNumber("Intake/lowerSensorReading", this.lowerSensor.getRange());
+		SmartDashboard.putNumber("Intake/upperSensorReading", this.upperSensor.getRange());
```
[the remark this answers is not in the document]
Comment (Lu-han-wang) [reply src/main/java/org/jmhsrobotics/frc2024/subsystems/intake/IntakeSubsystem.java]: deleted

**Comment by m10653** [comment]
Ready for merge?

**Review by m10653 — APPROVED** [review]

**Review by m10653 — APPROVED** [review]

**Pull request merged**